code/src/java/pcgen/persistence/lst/CampaignLoader.java

```diff
@@ -157,19 +155,6 @@ public void loadCampaignLstFile(URI filePath) throws PersistenceLayerException
 		{
 			// Check the campaign's prerequisites, generating errors if any are not met but proceeding
 			validatePrereqs(campaign.getPrerequisiteList());
-			List<String> copyright = campaign.getListFor(ListKey.SECTION_15);
```

Comment:
Why is the copyright portion removed?

Reply:
it doesn't actually do anything.  the list is retrieved for adding and then never used.  according to my search at least.  if there is a way it is being accessed i need to change my techniques for searching.

Reply:
I don't know that it is used - I'll search as well.  If it isn't used - might as well remove dead code.

Reply:
i'm using intellij's search functions if it helps

Reply:
Okay - looks good.  Nice work!